### PR TITLE
Add ID field to Node

### PR DESCRIPTION
**Pull request by fanick-ber**
this PR could be useful to solve issue #10 
It is a straightforward implementation to receive an array of the values to be able to identify the position in a tree.

### CODE REVIEW

File: src/index.jsx [hunk 1]
```diff
@@ -30,7 +30,9 @@ const tree = [
 
 class App extends Component {
   /* eslint-disable-next-line no-alert */
-  handleLeafClick = node => alert(`${node} clicked`);
+  handleLeafClick = node =>
+    /* eslint-disable-next-line no-alert */
+    alert(`${JSON.stringify(node)} clicked`);
```
Comment (helfi92) [inline src/index.jsx]: Remove the template string as well as "clicked":

```jsx
alert(JSON.stringify(node));
```

File: src/components/MuiTreeView/index.jsx [hunk 2]
```diff
@@ -14,6 +22,7 @@ const pickClassName = prop('className');
 /** Prop-type for a recursive data structure */
 const tree = {
   value: string.isRequired,
+  id: oneOfType[(string, number)],
```
Comment (helfi92) [inline src/components/MuiTreeView/index.jsx]: While you're at it, could you add comment to both `value` and `id`, please? Something like:

```jsx
// The node value.
value: string.isRequired,
// Optional node ID. Useful for when the node value is not unique.
id: oneOfType[(string, number)],
```

**Comment by fanick-ber** [comment]
Thanks a lot for your suggestions. I modified accordingly.

**Comment by fanick-ber** [comment]
I'm sorry, there was still a mistake with the prop-type. It's corrected in commit `776ddbe`

**Comment by helfi92** [comment]
`npm` is giving me trouble when publishing. Something may be down. I will give it another try in a bit.

**Comment by helfi92** [comment]
Published in `v3.0.2`.

**Pull request merged**